.travis.yml

```diff
@@ -61,14 +66,30 @@ script:
 after_success:
   - coveralls
 
-deploy:
-  provider: pypi
-  user: inveniosoftware
-  password:
-    secure: TODO:PYPISECUREHASH
-  distributions: "compile_catalog sdist bdist_wheel"
-  on:
-    tags: true
-    python: "2.7"
-    repo: inveniosoftware/invenio-app-ils
-    condition: $DEPLOY = true
+jobs:
+  include:
+    - stage: ui-test
+      language: node_js
+      node_js:
+        - 8
+      cache:
+        directories:
+          - node_modules
+      before_install:
+        - cd "./invenio_app_ils/ui/backoffice"
+      install:
+        - npm install
+      script:
+        - npm run test:ci
+    - stage: deploy
+      script: skip
+      deploy:
```

Comment:
I think we will never deploy this, we will deploy from a Python build, no?

Reply:
I moved the python deploy to a new stage so that it will deploy only if both backend and ui tests fail.